plugin/PySrc/mock_turtle.py

```diff
@@ -19,6 +21,7 @@
     tk.simpledialog = imp.new_module(tkinter_name)
     sys.modules[tkinter_name + '.simpledialog'] = tk.simpledialog
 
+import pyglet
```

Comment:
Replace `pyglet` with `pyglet_module` below, and then you won't need this import. This import will break if the user doesn't have pyglet installed.

Reply:
I think I've solved this now - had to define those classes inside a function body to avoid NameErrors. I've followed the matplot pattern - let me know if that was the right approach.